gcloud/bigquery/job.py

```diff
@@ -919,3 +931,278 @@ def _scrub_local_properties(self, cleaned):
                 assert dest_remote['projectId'] == self.project
                 dataset = self._client.dataset(dest_remote['datasetId'])
                 self.destination_table = dataset.table(dest_remote['tableId'])
+
+
+class _SyncQueryConfiguration(object):
+    """User-settable configuration options for synchronous query jobs."""
+    # None -> use server default.
+    _default_dataset = None
+    _max_results = None
+    _timeout_ms = None
+    _preserve_nulls = None
+    _use_query_cache = None
+
+
+class RunSyncQueryJob(_BaseJob):
```

Comment:
Maybe:  `RunSyncQuery` is actually an adjectival phrase, describing the kind-of `Job` this subclass represents.